static/sidebar-single/js/grid.js

```diff
@@ -148,7 +150,7 @@
       if (that.windowSize !== newSize) {
         that.windowSize = newSize;
         var isSmall = newSize === 'small';
-        that.itemsPerPage = isSmall ? 2 : (options.itemsPerPage || 6);
+        that.itemsPerPage = isSmall ? 2 : (options.items_per_page || 6);
```

Comment:
This should read:

**that.itemsPerPage = isSmall ? 2 : options.items_per_page;**

`items_per_page` should exists in the `defaults/` folder and it should have a default value set (6 i guess)